src/views/Request.vue

```diff
@@ -54,7 +59,9 @@
 		<LibresignTab v-if="showSidebar"
 			:prop-file="file"
 			:prop-name="file.name"
-			@close="showSidebar = false" />
+			@close="showSidebar = false"
```

Comment:
necessary fix the linter

run the follow commands to fix this check and others in this PR:
```
npm run lint:fix
npm run stylelint:fix
```